phidgets_high_speed_encoder/include/phidgets_high_speed_encoder/high_speed_encoder_ros_i.h

```diff
@@ -69,6 +69,8 @@ class HighSpeedEncoderRosI final
     // (Default=1) Number of "ITERATE" loops without any new encoder tick before
     // resetting the filtered average velocities.
     int speed_filter_idle_iter_loops_before_reset_;
+    Phidget_EncoderIOMode io_mode_;
+    int data_interval_;
```

Comment:
These don't need to be member variables, right? (see phidgets_motors for example)

I don't have a strong opinion on this, we can leave it as member variables if you prefer. I just try to keep the diff between the different phidgets small because there's a lot of duplicated code, and at some point somebody should refactor it (introducing a superclass or something), which will be easier the fewer differences we have between phidgets.